### ISSUE TITLE
Symbols and IntelliSense don't work at all

### ISSUE THREAD

**Issue by buu700** [Microsoft/vscode#6222]
- VSCode Version: 1.1.0
- OS Version: OS X 10.11.4

Steps to Reproduce:

1. Open any TypeScript file
2. Open symbols, find all references, go to definition, etc.

Expected Result:

It should work.

Actual Result:

No symbols, no references, etc.

---

Not sure what I might be doing wrong. I've tried uninstalling/reinstalling VS Code and deleting ~/Library/Application Support/Code, and nothing seems to work. This definitely used to work back when I started using VS Code a year or so ago, but some time in the last few months it stopped working for no apparent reason.

**Comment by dbaeumer** [comment]
@buu700 would you be able to share you project? Do you have a tsconfig.json file to configure the TS language server for your project. See also: https://code.visualstudio.com/docs/languages/typescript

**Comment by buu700** [comment]
Thanks Dirk. It seems like I'm able to consistently reproduce this with any arbitrary TypeScript file even outside the context of any project, and I confirmed with someone else working on the same project that he couldn't reproduce this issue while working on the project.

That said, an almost up-to-date version of the project is at https://github.com/cyph/cyph/tree/master/shared/js, and the current tsconfig.json is:

```
{
	"compileOnSave": true,
	"compilerOptions": {
		 "target": "es5",
		 "module": "system",
		 "moduleResolution": "classic",
		 "emitDecoratorMetadata": true,
		 "experimentalDecorators": true,
		 "rootDir": "."
	}
}
```

**Comment by buu700** [comment]
Also, as per that document, last night I tried configuring a task runner as well. This allowed me to successfully build the project and see the output in VS Code, but had no impact on my problem.

**Comment by dbaeumer** [comment]
@buu700 how is your OS X file system configured. Do you have it set to be case sensitive. We are aware of a problem with TS and case sensitive file systems under Mac that the TS team is working on fixing. This would explain why other team members on the project don't see the issue.

**Comment by buu700** [comment]
Ahh, got it, thanks! Yep, I do have it set to case-sensitive. Any idea what the general ETA or priority of that fix is?

**Comment by dbaeumer** [comment]
The TS issue is: https://github.com/Microsoft/TypeScript/issues/8139. It will be fixed in TS 2.0. The corresponding vscode issue is https://github.com/Microsoft/vscode/issues/5161 which also contains steps how to already make use of the fix in the tsserver.

Closing as dup of https://github.com/Microsoft/vscode/issues/5161

**Comment by buu700** [comment]
Sounds good, thanks for the info!
